W-14208671 get array count cv (#276)
mulesoft/docs-rpa: rpa-builder/6.4/modules/ROOT/pages/toolbox-variable-handling-get-array-count.adoc

number of elements is to be determined.

This array must be created beforehand using the _Workflow Based Variables_ element or
any action step that creates or returns an array.

The return array from the _Read from Excel File_ Action

> Step is only shown within the same transaction as a Pin Variable. To use the _Array Count_ in a later transaction, copy the return array into an array created beforehand in the Workflow Based Variables (see _Set_Array_Variable_).

Reviewer: ```suggestion
Step is shown only within the same transaction as a Pin Variable. To use the _Array Count_ in a later transaction, copy the return array into an array created beforehand in the Workflow Based Variables (see _Set_Array_Variable_).

```

//image:media\image1.png[Ein Bild, das Text enthält. Automatiscgenerierte Beschreibung,width=381,height=226]

> All arrays which are available within the Workflow show in the drop-down list next to _Array to get count of_.

Reviewer: ```suggestion
All arrays that are available within the Workflow are shown in the drop-down list next to _Array to get count of_.

```

These services are read from an array and checked off on a checklist.

> To test the Workflow, the _Workflow Based Variables_ were

Reviewer: ```suggestion
To test the Workflow, the _Workflow Based Variables_ are

```
